src/notation/internal/notationinteraction.cpp

```diff
@@ -5343,6 +5343,7 @@ void NotationInteraction::getLocation()
             score()->setPlayNote(true);
         }
         select({ e }, SelectType::SINGLE);
+        showItem(e);
```

Comment:
I actually would've thought the select function would have a flag to indicate you want it scrolled into view - that'd probably be the default expectation...